src/SamplesApp/SamplesApp.Shared/Strings/es-ES/Resources.resw

```diff
@@ -126,4 +126,23 @@
   <data name="PublisherDisplayName" xml:space="preserve">
     <value>Uno Platform</value>
   </data>
-</root>
\ No newline at end of file
+
+  <data name="TextBoxCut" xml:space="preserve">
+    <value>Cortar</value>
```

Comment:
These resources should ideally be part of Uno.UI itself. I think it could be possible to get most of them by querying the `ResourceLoader` in a running WinAppSDK app. I have tried that before here - https://gist.github.com/MartinZikmund/fd68639c0a887e7753b02ed41addbd3c ; maybe similar commands will be there for all of these options

Reply:
Yes, I think there should be one for each - see https://learn.microsoft.com/en-us/uwp/api/windows.ui.xaml.input.standarduicommandkind?view=winrt-22621

Reply:
Indeed, let's move to uno.ui.